js/test/root/teststrings.js

```diff
@@ -3670,5 +3691,25 @@ module.exports.teststrings = {
 
         test.equal(str.formatChoice(1.7), "Default items");
         test.done();
+    },
+    testStringFormatChoiceDecimal_ca_AD: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("ca-AD");
```

Comment:
Should this be ca-ES?

Reply:
This is correct. I additional added test cases for ca-ES